mingw-w64-git/PKGBUILD

```diff
@@ -73,12 +73,14 @@ sha256sums=('SKIP'
             '6d83e1cb1acdb6eb1f2d5cb9299298e57680f5ca43d43c3e67c9da17f21b9b01')
 
 USE_ASCIIDOCTOR="YesPlease"
+COMPAT_CFLAGS=
 
 if test -z "$WITHOUT_PDBS"
 then
 	pkgname+=("${MINGW_PACKAGE_PREFIX}-${_realname}-pdb")
 	makedepends+=("${MINGW_PACKAGE_PREFIX}-cv2pdb")
 	options+=('!strip')
+	COMPAT_CFLAGS="-gdwarf-4 -gstrict-dwarf" # cv2pdb cannot really handle DWARF5 yet
```

Comment:
Oh wow, I had completely forgotten about that. As of https://github.com/rainers/cv2pdb/pull/69, that setting should not be needed any longer.

However, that's not _your_ problem to solve, i.e. this commit is fine as-is.